apps/server/src/shared/domain/domainobject/board/card.do.ts

```diff
@@ -0,0 +1,39 @@
+import { EntityId } from '@shared/domain/types';
+import { TextElement } from './text-element.do';
+
+export class Card implements CardProps {
+	id: EntityId;
+
+	title: string;
```

Comment:
Is it really the card's title or the content's title?

Reply:
I guess one could argue that this is the title of the content that the card represents.
But since there is a title on almost every object on every level of the tree, for ease of use and programming I think it makes sense to have the same property for the title on every layer.

The DOs are not fleshed out yet, but they will become a composable structure with common interface or superclass.